lib/rubocop/cop/metrics/cyclomatic_complexity.rb

```diff
@@ -13,19 +13,53 @@ module Metrics
       # operator (or keyword and) can be converted to a nested if statement,
       # and ||/or is shorthand for a sequence of ifs, so they also add one.
       # Loops can be said to have an exit condition, so they add one.
+      # Blocks that are calls to builtin iteration methods
+      # (e.g. `ary.map{...}) also add one, otherse are ignored.
```

Comment:
Might also be a good idea to add a couple of code examples with inline comments showing the complexity calculation (e.g. +1, ignored, etc).

Reply:
Example added 👍